src/chrome/content/rules/AOL.xml

```diff
@@ -190,12 +184,6 @@
 
 	<target host="expapi.oscar.aol.com" />
 
-		<!--	Added by AOL, required for http logout.
-								-->
-		<exclusion pattern="^http://my\.screenname\.aol\.com/_cqr/logout/" />
-
-			<test url="http://my.screenname.aol.com/_cqr/logout/" />
-
```

Comment:
I feel a bit uncomfortable with that, since it was added by the AOL team itself.